config/locales/views/talk/show/en.yml

```diff
@@ -0,0 +1,7 @@
+# See https://github.com/svenfuchs/rails-i18n/tree/master/rails%2Flocale for starting points.
+
+en:
+  talk:
+    show:
+      welcome: "Hello Everyone! Welcome to Publiclab Talk Pad"
```

Comment:
I think it's a bit long -- could we use just the first line? We can do:

> [Public Lab]() | Talk page for [Page name]() --  This page allows for synchronized editing by multiple people as you type. [Learn more]()

And link to a wiki page at `/wiki/talk-pages` there, so people can read the rest of your suggestions?